src/main/java/delight/rhinosandox/internal/RhinoSandboxImpl.java

```diff
@@ -6,237 +6,304 @@
 import delight.rhinosandox.internal.SafeClassShutter;
 import delight.rhinosandox.internal.SafeContext;
 import delight.rhinosandox.internal.SafeWrapFactory;
+
+import java.io.IOException;
+import java.io.Reader;
 import java.lang.reflect.Method;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
+
 import org.eclipse.xtext.xbase.lib.Exceptions;
-import org.mozilla.javascript.Context;
-import org.mozilla.javascript.ContextFactory;
-import org.mozilla.javascript.Scriptable;
-import org.mozilla.javascript.ScriptableObject;
+import org.mozilla.javascript.*;
```

Comment:
Could we leave it here as explicit imports rather than using the wildcard import?